docs/annexes/annex-06-pid-rulebook.md

```diff
@@ -491,16 +517,110 @@ If other CDDL representation types will be used in future versions of
 this document, the corresponding rules for encoding them with JSON will
 be added here.
 
-##### 2.5.3.2 Further stipulations
-
-Data elements in Table 1 and Table 2 SHALL be released only in a VP
-Token, as specified in \[OpenID4VP\]. This means that these data
-elements SHALL be signed by the PID Provider, not by the Wallet
-Instance.
+##### 2.6.3.2 Data Element Identifer to Claim Mapping
+
+SD-JWT encoded PID attestations SHALL use claim names registered in the JSON Web Token Claims Registry \[IANA-JWT-Claims\]. Table 3 below maps the data element identifiers from Table 1 and Table 2 above to the corresponding claim names. 
+
+Note that a hierarchical claim name structure can be used in SD-JWT VC encoded PID attestations as SD-JWT allows for individual selective disclosure of objects and their properties. A hierarchical claim name structure is indicated by the notation `parent/child` in Table 3.
+
+Data element identifiers not listed in Table 3 are used as defined above.
+
+| Data Element Identifier | Claim Name | Note |
+|---|---|---|
+| `birth_date` | `birthdate` |  |
+| `age_over_18` | `age_equal_or_over/18` | `18` property name is a string. *)|
+| `age_over_NN` | `age_equal_or_over/NN` | NN property name is a string. *) |
+| `family_name_birth` | `birth_family_name` | |
+| `given_name_birth` | `birth_given_name` | |
+| `birth_place` | `place_of_birth/locality` | *) |
+| `birth_country` | `place_of_birth/country` | |
+| `birth_state` | `place_of_birth/region` | |
+| `birth_city` | `place_of_birth/locality` | |
+| `resident_address` | `address/formatted` | |
+| `resident_country` | `address/country` | |
+| `resident_state` | `address/region` | |
+| `resident_city` | `address/locality` | |
+| `resident_postal_code` | `address/postal_code` | |
+| `resident_street` | `address/street_address` | *) |
```

Comment:
Is there a reason why Table 3 uses slash `/` as the hierarchical delimiter? Dot notation `.` is more commonly used in JSON contexts. I think `address.street_address` looks more familiar than `address/street_address`.

Reply:
Either works for me - I'll modify to `.`